helper/reports.go

```diff
@@ -80,3 +80,21 @@ func ProcessDownloadReportCSV(reportID int, outputFilePath string) error {
 	return err
 
 }
+
+func ProcessCreateReport(fileID int) (reportID int, err error) {
+	ctx := context.Background()
+	client := getClient()
+	reportID, err = client.Reports.CreateReport(ctx, fileID)
+	if err != nil && err.Error() == "400 Bad Request" {
+		results, err := client.Reports.List(ctx, fileID)
+		if err != nil {
+			return 0, err
+		}
+		return results[0].ID, nil
```

Comment:
Getting non-empty report list can't be guaranteed. The scenario where report list is empty should ideally be handled here.